docs/sources/mimir/manage/tools/mimir-continuous-test.md

```diff
@@ -50,7 +56,7 @@ Mimir-continuous-test requires the endpoints of the backend Grafana Mimir cluste
 - Set `-tests.smoke-test` to run the test once and immediately exit. In this mode, the process exit code is non-zero when the test fails.
 
 {{< admonition type="note" >}}
-You can run `mimir-continuous-test -help` to list all available configuration options.
+You can run `mimir -help` to list all available configuration options. All configuration options for mimir-continuous-test will begin with `tests`.
```

Comment:
nit: "will" changes the tense and removing it keeps us in the present which is more accurate because we are discussing what is true and not will be true in the future.

```suggestion
You can run `mimir -help` to list all available configuration options. All configuration options for mimir-continuous-test begin with `tests`.
```